src/modules/global_css/index.js

```diff
@@ -28,14 +17,22 @@ class GlobalCSSModule {
             defaultValue: false,
             description: 'Enable Twitch\'s dark theme'
         });
-        settings.on('changed.darkenedMode', () => this.loadDark());
-        this.loadDark();
+
+        settings.on('changed.darkenedMode', isDark => this.darkChange(isDark));
+        watcher.on('dark.change', isDark => this.darkChange(isDark));
+
+        this.darkSet(settings.get('darkenedMode'));
+    }
+
+    darkSet(isDark) {
+        const connectRoot = twitch.getConnectRoot();
+        if (!connectRoot) return;
+        connectRoot._context.store.dispatch({type: 'core.ui.THEME_CHANGED', theme: (isDark ? 1 : 0)});
```

Comment:
type should really be a constant atop the file.. theme should use a constant too (example in my other comment).